src/Paket.Core/ScriptGeneration.fs

```diff
@@ -166,7 +169,7 @@ module ScriptGeneration =
           let scriptFile = getScriptFile groupName package.Name
         
           let groupName = getGroupNameAsOption groupName
-          let dependencies = package.Dependencies |> Seq.map (fun (depName,_,_) -> knownIncludeScripts.[depName])
+          let dependencies = package.Dependencies |> Seq.choose (fun (depName,_,_) -> knownIncludeScripts.TryFind depName)
```

Comment:
this makes it so that we only generate #loads for dependencies that we actually generate